sdcm/tester.py

```diff
@@ -1019,6 +1020,37 @@ def create_ks(self, session, name, rf):
                                      options))
         session.execute('USE %s' % name)
 
+    def is_keyspace_in_cluster(self, session, keyspace_name):
+        query_result = session.execute("SELECT * FROM system_schema.keyspaces;")
+        keyspace_list = [row.keyspace_name.lower() for row in query_result.current_rows]
+        return keyspace_name.lower() in keyspace_list
+
+    def wait_validate_keyspace_existence(self, session, keyspace_name, timeout=180, step=5):
+        text = "waiting for the keyspace {} to be created in the cluster".format(keyspace_name)
+        does_keyspace_exist = wait.wait_for(func=self.is_keyspace_in_cluster, step=step, text=text, timeout=timeout,
+                                            session=session, keyspace_name=keyspace_name)
+        return does_keyspace_exist
+
+    def create_keyspace_generic(self, keyspace_name, replication_strategy, replication_factor):
```

Comment:
`create_keyspace_generic` -> `create_keyspace`

Reply:
add docstring that explains how to use `replication_factor` in case of multi dc

Reply:
move `replication_strategy=None` as last argument and add select default strategy dynamically